jvm/src/vyxal/gen/GenerateDocs.scala

```diff
@@ -57,4 +59,46 @@ private object GenerateDocs:
     SugarMap.trigraphs
       .map { case (key, value) => s"$key -> $value" }
       .mkString("", "\n", "\n")
+
+  def elementTable(): String =
+    val header = "| Symbol | Name | Keywords | Arity | Vectorises | Overloads |"
+    val divider = "| --- | --- | --- | --- | --- | --- |"
+    val contents = StringBuilder()
+    val formatOverload = (overload: String) =>
```

Comment:
```suggestion
    def formatOverload(overload: String) =
```

Reply:
See the other conversation for why I wanted to intentionally use a lambda.